Assets/Scripts/Items/PopupInventoryItem.cs

```diff
@@ -57,18 +53,17 @@ public class PopupInventoryItem : MonoBehaviour {
     }
 
     private IEnumerator AnimateSprite() {
-        int animationLoop = 0;
+        itemRendered = false;
```

Comment:
We have to set this false as soon as we start this coroutine, b/c otherwise it's going to keep being called every frame until the first instance has completed (after the animation has finished).